tck/src/main/java/org/eclipse/microprofile/openapi/apps/airlines/resources/ZepplinResource.java

```diff
@@ -71,4 +79,28 @@ public Response headZepplin() {
     public Response getZepplin() {
         return Response.ok().build();
     }
+
+    @POST
+    @APIResponse(responseCode = "200", description = "Review posted")
+    @APIResponse(responseCode = "404", description = "Review not found")
+    @Operation(summary = "Post by Zepplin", operationId = "postZepplin")
+    @Produces("text/plain")
+    @SecurityRequirement(name = "mutualTLSScheme", scopes = "zepplinScope")
+    @Consumes("multipart/form-data")
+    public Response postZepplin(
+            @RequestBody(description = "Record of a new user to be created in the system.", required = true,
+                         content = @Content(mediaType = "multipart/form-data",
+                                            schema = @Schema(name = "testUser", type = SchemaType.OBJECT,
+                                                             maxProperties = 1024, minProperties = 1, required = true,
+                                                             implementation = User.class),
+                                            encoding = {
+                                                    @Encoding(name = "allergies",
+                                                              allowReserved = true, explode = true,
+                                                              style = "pipeDelimited"),
+                                                    @Encoding(name = "specialRequests",
+                                                              allowReserved = true, explode = true,
+                                                              style = "spaceDelimited")
```

Comment:
Every `@Encoding.name` must exist as a named property within the schema in order to satisfy the following requirement from the [Media Type Object](https://spec.openapis.org/oas/v3.1.0.html#media-type-object) encoding field:

> The key, being the property name, MUST exist in the schema as a property.

Reply:
It might be better to use a new object to define the schema, rather than `User`, or to use `@Schema.properties` to define the properties needed instead of an implementation object.